tools/pi_db_tools/pi_database_splitter.py

```diff
@@ -0,0 +1,179 @@
+# => python3.3 required!

```

Comment:
Aha, but then we should also just write `$__tool_directory/script.py` instead of with `python` or `python3`.